container-search/src/test/java/com/yahoo/search/significance/test/SignificanceSearcherTest.java

```diff
@@ -0,0 +1,119 @@
+// Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.search.significance.test;
+
+import com.yahoo.component.chain.Chain;
+import com.yahoo.language.Language;
+import com.yahoo.language.significance.SignificanceModel;
+import com.yahoo.language.significance.SignificanceModelRegistry;
+import com.yahoo.language.significance.impl.DefaultSignificanceModelRegistry;
+import com.yahoo.prelude.query.AndItem;
+import com.yahoo.prelude.query.WordItem;
+import com.yahoo.search.Query;
+import com.yahoo.search.Result;
+import com.yahoo.search.searchchain.Execution;
+import com.yahoo.search.significance.SignificanceSearcher;
+import org.junit.jupiter.api.Test;
+
+import java.nio.file.Path;
+import java.util.HashMap;
+
+import static com.yahoo.test.JunitCompat.assertEquals;
+
+/**
+ * Tests significance term in the search chain.
+ *
+ * @author MariusArhaug
+ */
+public class SignificanceSearcherTest {
+    SignificanceModelRegistry significanceModelRegistry;
+    SignificanceSearcher searcher;
+
+    public SignificanceSearcherTest() {
+        HashMap<Language, Path> map = new HashMap<>();
+        map.put(Language.ENGLISH, Path.of("src/test/java/com/yahoo/search/significance/model/en.json"));
+
+        significanceModelRegistry = new DefaultSignificanceModelRegistry(map);
+        // TODO change to mock
+        searcher = new SignificanceSearcher(significanceModelRegistry);
+    }
+
+    private Execution createExecution() {
+        return new Execution(new Chain<>(searcher), Execution.Context.createContextStub());
+    }
+
+    @Test
+    void testSimpleSignificanceValue() {
```

Comment:
Rename the method to make it clear that it's the query that's 'simple', not the significance value.

Reply:
Renamed methods

Reply:
Did you forget to push the commit? 😄